app/models/create_api_action.rb

```diff
@@ -0,0 +1,3 @@
+class CreateApiAction < ApiAction
+    
```

Comment:
@Pralish are these classes: `CreateApiAction` `DestroyApiAction` and `ErrorApiAction` being used?

Reply:
```
  has_many :create_api_actions, dependent: :destroy
  accepts_nested_attributes_for :create_api_actions, allow_destroy: true

  has_many :show_api_actions, dependent: :destroy
  accepts_nested_attributes_for :show_api_actions, allow_destroy: true

  has_many :update_api_actions, dependent: :destroy
  accepts_nested_attributes_for :update_api_actions, allow_destroy: true

  has_many :destroy_api_actions, dependent: :destroy
  accepts_nested_attributes_for :destroy_api_actions, allow_destroy: true

  has_many :error_api_actions, dependent: :destroy
  accepts_nested_attributes_for :error_api_actions, allow_destroy: true
  ```
  
  @donrestarone , we do use them.  These classes inherit from ApiAction